### PR TITLE
Setup architecture and add menu and slots

### PR DESCRIPTION
**Pull request by ckeisc43rd-cjtsai**

### CODE REVIEW

File: app/controllers/slots_controller.rb
Comment (AaronWu-train) [inline app/controllers/slots_controller.rb:1]: Please add TODO commands on empty functions. etc. show

File: app/views/layouts/application.html.erb
Comment (AaronWu-train) [inline app/views/layouts/application.html.erb:1]: html file isn't formated

File: test/controllers/menu_controller_test.rb
Comment (AaronWu-train) [inline test/controllers/menu_controller_test.rb:1]: Please remove unnecessary tests, or add TODO comments.

File: test/controllers/slot_controller_test.rb
Comment (AaronWu-train) [inline test/controllers/slot_controller_test.rb:1]: Please remove unnecessary tests, or add TODO comments.

File: test/fixtures/matches.yml
Comment (AaronWu-train) [inline test/fixtures/matches.yml:1]: Please remove unnecessary tests, or add TODO comments.

File: test/fixtures/singles.yml
Comment (AaronWu-train) [inline test/fixtures/singles.yml:1]: Please remove unnecessary tests, or add TODO comments.

File: test/fixtures/slots.yml
Comment (AaronWu-train) [inline test/fixtures/slots.yml:1]: Please remove unnecessary tests, or add TODO comments.

**Review by AaronWu-train — CHANGES_REQUESTED** [review]
- Please add TODO comments
- Format codes using prettier or other formatter
- Readmd.md requires more detailed instructions

**Comment by ckeisc43rd-cjtsai** [comment]
Resolved questions listed above

**Review by AaronWu-train — APPROVED** [review]

**Comment by AaronWu-train** [comment]
@ckeisc43rd-cjtsai Please add the prittierrc file and commands to yarn

**Pull request merged**